src/main/java/game/RacingGame.java

```diff
@@ -0,0 +1,34 @@
+package game;
+
+import game.ui.RacingGameInputView;
+import game.ui.RacingGameResultView;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.IntStream;
+
+public class RacingGame {
+    private final MovePolicy movePolicy;
+    private final RacingGameInputView racingGameInputView;
+    private final RacingGameResultView racingGameResultView;
+
+    public RacingGame(MovePolicy movePolicy, RacingGameInputView racingGameInputView, RacingGameResultView racingGameResultView) {
+        this.movePolicy = movePolicy;
+        this.racingGameInputView = racingGameInputView;
+        this.racingGameResultView = racingGameResultView;
+    }
+
+    public void play() {
+        int numberOfCar = racingGameInputView.getNumberOfCar();
+        int time = racingGameInputView.getTime();
+        List<Car> carList = new ArrayList<>();
+        IntStream.range(0, numberOfCar).forEach(i -> carList.add(new Car(i, 0, movePolicy)));
+
+        for (int i = 0; i < time; i++) {
+            racingGameResultView.printRound(i+1);
```

Comment:
이항 연산자 사이에는 공백으로 구분하는 컨벤션이 있습니다 :)
for문에서 잘 지키셨는데, 이 부분만 놓치셨네요
```suggestion
        IntStream.range(0, numberOfCar)
                 .forEach(i -> carList.add(new Car(i, 0, movePolicy)));

        for (int i = 0; i < time; i++) {
            racingGameResultView.printRound(i + 1);
```